Upstream: yes
Reason: fix building of tests

From e3b4b3660fe3e217e68516e2222551787bffda5c Mon Sep 17 00:00:00 2001
From: Aleix Pol <aleixpol@kde.org>
Date: Fri, 8 Nov 2013 03:13:13 +0100
Subject: [PATCH] Compile everything

Albert, this is asking you to be a library ;)

CCMAIL: albertvaka@gmail.com
---
 tests/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index a9a8bad..b559abf 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -24,6 +24,6 @@ set(kdeconnect_libraries
 )
 
 #NetworkPackage
-kde4_add_unit_test(kdeconnect_tests ../kded/networkpackage.cpp ../kded/filetransferjob.cpp networkpackagetests.cpp)
+kde4_add_unit_test(kdeconnect_tests ../kded/networkpackage.cpp ../kded/kdebugnamespace.cpp ../kded/filetransferjob.cpp networkpackagetests.cpp)
 target_link_libraries(kdeconnect_tests ${kdeconnect_libraries})
 
-- 
1.8.4.2

